### PR TITLE
Fixed b6mini permission error preventing charging, added QoL udev rules.

### PR DESCRIPTION
**Pull request by ThePakkrat**
b6mini device permission error resolved. Can now charge batteries, but still cannot select individual chargers. Added Documentation folder with .rules example (udev) to let user run "python test.py" (to start charging a batterry) without requiring sudo.

Commented out section of getChargeInfo function in Device.cc that kept throwing an error.

### CODE REVIEW

File: libraries/Charger.cpp [hunk 1]
```diff
@@ -12,7 +12,7 @@
 // ██      ███████ ███████ ██████  ██   ███ █████   ██████
 // ██      ██   ██ ██   ██ ██   ██ ██    ██ ██      ██   ██
 //  ██████ ██   ██ ██   ██ ██   ██  ██████  ███████ ██   ██
-
+::
```
Comment (ryanpdx) [inline libraries/Charger.cpp:15]: ??

File: documentation/99-b6mini.rules [hunk 2]
```diff
@@ -0,0 +1,2 @@
+SUBSYSTEM=="usb", ATTRS{idVendor}=="0000", ATTRS{idProduct}=="0001", MODE:="666", GROUP="plugdev"
+KERNEL=="hidraw*", ATTRS{idVendor}=="0000", ATTRS{idProduct}=="0001", MODE:="666", GROUP="plugdev"
```
Comment (ryanpdx) [inline documentation/99-b6mini.rules:2]: Add the instructions on how to use this to the readme

File: software/playground.py [hunk 3]
```diff
@@ -11,4 +11,39 @@
...
 print(bb.cellVolts)
-print(bb.cellTemps)
\ No newline at end of file
+print(bb.cellTemps)
+
+#blame Lev below this line
+from datetime import datetime
+import csv
+
+#function takes in timestamp and voltage, writes values to csv file, "a" appends whereas "w" clears file first
```
Comment (ryanpdx) [inline software/playground.py:20]: Docstrings should be  under function def and block comments

File: software/playground.py [hunk 4]
```diff
@@ -11,4 +11,39 @@
...
+        pass
+
+#function makes a timestamp
+def stamper():
+    dt = datetime.now()
+    iso_date = dt.isoformat()
+    return iso_date
+
+check = input("Do you want to clear the csv file, good sir? \nY/n: ")
+if (check.upper == 'Y'):
```
Comment (ryanpdx) [inline software/playground.py:39]: In Python parentheses are not need in if statements

**Review by ryanpdx — CHANGES_REQUESTED** [review]
Remove the python bytecode files (the `*.pyc` files) and add a line to the main `.gitignore` file to ignore them. Those are generate files that should not be tracked with git.

File: software/playground.py (hunk 3 above)
Comment (ThePakkrat) [reply software/playground.py]: If this means that the comments for each function should be under the function while non-function-specific comments should be above, then we should be good now.

File: software/playground.py (hunk 4 above)
Comment (ThePakkrat) [reply software/playground.py:46]: I like the parentheses for readability's sake - if it is bad to have them though, I will remove them. All other requested changes have been implemented and committed.

**Comment by ThePakkrat** [comment]
Should be good for a second look now.

File: README.md [hunk 5]
```diff
@@ -13,6 +13,13 @@ V1.0 of the PCB layout:
 - Initialize libb6 submodule: `$ git submodule update --init`
 - Build Python extension for libb6: `$ make -C libraries`
 
+### Udev Setup
+Copy the .rules file (from the Documentation folder) into the folder on your computer which contains rules files for udev.
+
+For me it is: Computer/etc/udev/rules.d
```
Comment (ryanpdx) [inline README.md:19]: You can just say `/etc/udev/rules.d/`

Leading `/` signifies an absolute path

File: software/playground.py (hunk 4 above)
Comment (ryanpdx) [reply software/playground.py:46]: Up to you, it is a PEP standard to not include them when not needed. PEP standards are more like guidelines than rules.

File: software/playground.py [hunk 6]
```diff
@@ -11,4 +11,46 @@
...
+#blame Lev below this line
+from datetime import datetime
+import csv
+
+def write_to_csv(date, voltage):
+"""
+function takes in timestamp and voltage, 
+writes values to csv file, 
+"a" appends whereas "w" clears file first
+"""
```
Comment (ryanpdx) [inline software/playground.py:25]: Should be tabbed over to same indent as function code

File: README.md (hunk 5 above)
Comment (ThePakkrat) [reply README.md:19]: Good to know!

File: software/playground.py (hunk 6 above)
Comment (ThePakkrat) [reply software/playground.py]: Also good to know.

**Comment by ThePakkrat** [comment]
Should be good for third look now.

**Review by ryanpdx — APPROVED** [review]
LGTM!

**Comment by ThePakkrat** [comment]
Don't delete my branch though.

**Pull request merged**